package.json

```diff
@@ -57,17 +59,18 @@
     "underscore": "1.8.3"
   },
   "devDependencies": {
+    "deep-equal": "~1.0.1",
     "grunt": "~1.0.1",
     "grunt-fh-build": "~2.0.0",
     "istanbul": "0.4.3",
+    "license-reporter": "github:bucharest-gold/license-reporter#a58b9b570414fba843e4eefad0737595535f7a03",
```

Comment:
if we install license reporter in the `preshrinkwrap` stage do we still need it here? Or is this to guarantee that we install the correct version?

Reply:
Yeah, it gets the version from here. We're discussing on the license-reporter repo to see if we can make this cleaner by not including the devDependencies license info, even if they're present in node_modules -- similar to how shrinkwrap just includes production dependencies; so in future, the preshrinkwrap and postshrinkwrap hooks might be cleaner.